Cargo.toml

```diff
@@ -99,7 +102,15 @@ bitflags = "2.6"
 thunderdome = "0.6"
 crossbeam-utils = "0.8"
 fixed-resample = { version = "0.4.2", default-features = false }
+firewheel-macros = { path = "crates/firewheel-macros", version = "0.1.0" }
 
 # Optimize all dependencies in debug builds:
 [profile.dev.package."*"]
 opt-level = 2
+
+[dev-dependencies]
+criterion = "0.5"
+
+[[bench]]
+name = "core"
+harness = false
```

Comment:
This benchmarking is not critical, but may help guide future developments of the diffing API.